Source/RunActivity/Viewer3D/Viewer.cs

```diff
@@ -992,6 +1012,22 @@ void HandleUserInput(ElapsedTime elapsedTime)
             }
             if (UserInput.IsPressed(UserCommand.DisplaySwitchWindow)) if (UserInput.IsDown(UserCommand.DisplayNextWindowTab)) SwitchWindow.TabAction(); else SwitchWindow.Visible = !SwitchWindow.Visible;
             if (UserInput.IsPressed(UserCommand.DisplayTrainOperationsWindow)) if (UserInput.IsDown(UserCommand.DisplayNextWindowTab)) TrainOperationsWindow.TabAction(); else { TrainOperationsWindow.Visible = !TrainOperationsWindow.Visible; if (!TrainOperationsWindow.Visible) CarOperationsWindow.Visible = false; }
+
+            if (UserInput.IsPressed(UserCommand.DisplayTrainCarOperationsWindow))
+            {
+                if (UserInput.IsDown(UserCommand.DisplayNextWindowTab))
+                    TrainCarOperationsWindow.TabAction();
+                else
```

Comment:
hi Mauricio (@mbm-OR),

I think this `TrainCarOperationsWindow.TabAction(); ` is of no use. First of all there is no TabAction for TrainCarOperationsWindow. Secondly the UserCommand.DisplayNextWindowTab is only True when the Shift button is pressed. But Shift+F9 fires the "Display Train Dpu Window". So this TabAction code never gets fired.

Shall I remove it? 

I have more or less the same problem with UserCommand.DisplayTrainDpuWindow. Which does not pop up when activated from my web based Switch Panel. Trying to fix it gave a merge conflict with this code. I would like to fix this also in this PR.

Anyway complicated, I hope you understand.

Reply:
Hi Sierben, ( @sweiland-openrails ) 

> Shall I remove it?
> 
> I have more or less the same problem with UserCommand.DisplayTrainDpuWindow. Which does not pop up when activated from my web based Switch Panel.

All unused TabAction() have been removed. 
Your web based Switch Panel now seems to work with the UserCommand.DisplayTrainDpuWindow.